src/mcts/search.cc

```diff
@@ -1126,7 +1146,12 @@ int SearchWorker::PrefetchIntoCache(Node* node, int budget) {
 
 // 4. Run NN computation.
 // ~~~~~~~~~~~~~~~~~~~~~~
-void SearchWorker::RunNNComputation() { computation_->ComputeBlocking(); }
+void SearchWorker::RunNNComputation() {
+  if (params_.GetMaxConcurrentSearchers() != 0) {
```

Comment:
Why here and not e.g. in the end of GatherMinibatch?

Reply:
The difference is whether Prefetch can happen in parallel or not.  I kind of think prefetch being single thread at a time also makes sense.